### ISSUE TITLE
Código

### ISSUE THREAD

**Issue by fabriciomrkonell** [JoeVogel/Arduino-LineFollowerRobot#1]
Sério que é só isso?

**Comment by JoeVogel** [comment]
Sim. A dificuldade aqui não é o código e sim entender exatamente o que cada variável faz. O PID trabalha diferente em cada robo e deve ser precisamente ajustado. Isso é difícil. Provavelmente quando você utilizar o código no inicio o robo será um tremendo idiota, ai tem que ir acertando a regulagem das variáveis. Te explico melhor depois no laboratório.
